### PR TITLE
Update TORCH_XLA_USER_GUIDE.md

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: TORCH_XLA_USER_GUIDE.md [hunk 1]
```diff
@@ -55,6 +55,10 @@ sudo apt-get install libopenblas-dev"
 
 gcloud compute tpus tpu-vm scp params_70b.json ${TPU_NAME}:params.json --zone ${ZONE} --project ${PROJECT_ID} --worker=all
 
+gcloud compute tpus tpu-vm ssh ${TPU_NAME} --zone ${ZONE} --project ${PROJECT_ID} --worker=all --command="pip install torch_xla[tpu] -f https://storage.googleapis.com/libtpu-releases/index.html"
+
+gcloud compute tpus tpu-vm ssh ${TPU_NAME} --zone ${ZONE} --project ${PROJECT_ID} --worker=all --command="mv params.json llama/"
```
Comment (alanwaketan) [inline TORCH_XLA_USER_GUIDE.md]: what's this? Where is the params.json?

Comment (ManfeiBai) [reply TORCH_XLA_USER_GUIDE.md]: command above will copy [`params_7b.json`](https://github.com/pytorch-tpu/llama/blob/llama2-google-next-inference/TORCH_XLA_USER_GUIDE.md#prerequisites) into TPUVM as `params.json`, but path is outside of [`llama`](https://github.com/pytorch-tpu/llama/blob/llama2-google-next-inference/TORCH_XLA_USER_GUIDE.md#commands-to-run-llama2-using-xlatpu-tpu-v4) repo, so add this command to move `params.json` into `llama` repo

Comment (alanwaketan) [reply TORCH_XLA_USER_GUIDE.md]: But why moving it into the Llama repo? It looks like this path is from the command line argument. https://github.com/pytorch-tpu/llama/blob/6c7fe276574e78057f917549435a2554000a876d/llama/generation.py#L85

Comment (ManfeiBai) [reply TORCH_XLA_USER_GUIDE.md]: `params.json`'s path is outside of `llama/`, and the following command would `cd $HOME/llama` first before execute and use `params.json` directly from current path

**Review by alanwaketan — APPROVED** [review]

**Pull request merged**